Casks/p/piezo.rb

```diff
@@ -1,7 +1,13 @@
 cask "piezo" do
-  version "1.8.2"
   sha256 :no_check
 
+  on_ventura :or_older do
+    version "1.8.2"
+  end
+  on_sonoma :or_newer do
+    version "1.9.0"
+  end
+
```

Comment:
It seems that the system version in the livecheck url indicates the mac os version in numbers. Most recent sonoma version is 14.4.0 and so it should be separated. The version parameter do nothing I guess. The later livecheck should be deleted if you apply this change.

```suggestion
  on_ventura :or_older do
    version "1.8.2"

    livecheck do
      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=1231&bundleid=com.rogueamoeba.Piezo&platform=osx"
      strategy :sparkle
    end
  end
  on_sonoma :or_newer do
    version "1.9.0"

    livecheck do
      url "https://rogueamoeba.net/ping/versionCheck.cgi?format=sparkle&system=1440&bundleid=com.rogueamoeba.Piezo&platform=osx"
      strategy :sparkle
    end
  end
```

Reply:
Btw, sonoma with 14.3.1 will get the wrong version, so maybe it looks better to remain it until next major version update?

Reply:
Yeah, I just wasn't sure if someone would come here and know an incantation to actually pick out 14.4+ within a cask.

Reply:
We don't support scoping to minor/patch versions of macOS. It's presumed that users are on the latest minor/patch version.

Reply:
@p-linnane Is there a way to *obtain* that number, so it can be slotted into the livecheck URL?

Reply:
Not reliably to my knowledge. @Bo98 any ideas if there's a clever way to handle this?

Reply:
`MacOS.full_version` exists but it is probably not what you want. We don't store information on the latest patch release per major OS.

Reply:
The version that is in the `livecheck` won't affect what version is installed anyway. The new version will be installed on any machine running Sonoma.